test/extended/openstack/machine.go

```diff
@@ -95,10 +119,14 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] Machine", func()
 			})
 
 			g.By(fmt.Sprintf("Comparing Machine %q with instance %q: root volume", machineName, instance.Name), func() {
-				if instance.Image["id"] != nil {
-					instanceImage, err := images.Get(computeClient, fmt.Sprintf("%v", instance.Image["id"])).Extract()
-					o.Expect(err).NotTo(o.HaveOccurred())
-					o.Expect(instanceImage.Name).To(o.Equal(machineImage), "Image not matching for instance %q", instance.Name)
+				for _, vol := range instance.AttachedVolumes {
+					v, err := volumes.Get(computeClient, vol.ID).Extract()
```

Comment:
This is failing with:
```
fail [github.com/openshift/openstack-test/test/extended/openstack/machine.go:124]: Error fetching Openstack volumes for Machine "k0r4ltht-a50fa-slrwh-worker-0-ksw9k"
Unexpected error:
    <gophercloud.ErrDefault404>: 
    Resource not found: [GET https://compute.rdo.mtl2.vexxhost.net/v2.1/volumes/f35ae143-8c2d-4507-8c6b-74729ca6b550], error message: {"message": "The resource could not be found.<br /><br />\n\n\n", "code": "404 Not Found", "title": "Not Found"}
```
maybe a timeout?

Reply:
I Saw what and I wondered if there was  a problem with the code